### ISSUE TITLE
Cannot ignore multiple parameters in JSLint.NET

### ISSUE THREAD

**Issue by dlynchcodes** [benquarmby/jslintnet#33]
Hi there, I'm using Semantic UI and one of the function callbacks requires three parameters and in this particular usage I am only interested in the third parameter.

```
self.ClearVariation = function(value, text, $choice) {
//code
};
```

In this instance I am only interested in `$choice`
I can change `value` to `ignore` and JSLint will ignore this, but changing `text` to `ignore1` or some derivation of that does not. 

I am using Visual Studio 2015 if that helps.
How should I properly suppress this warning?
